data/en/ukis_0001.json

```diff
@@ -0,0 +1,4557 @@
+                            "title": "What was the estimated expenditure on acquisitions of Research & Development for the calendar year <em>2018 only</em>?",
+                            "guidance": {
+                                "content": [{
+                                        "description": "<strong>Include:</strong>"
+                                    },
+                                    {
+                                        "list": [
+                                            "internal costs and purchases from outside the business"
+                                        ]
+                                    }
+                                ]
+                            },
+                            "description": "",
+                            "type": "General",
+                            "answers": [{
+                                "id": "answer3171",
+                                "mandatory": false,
+                                "type": "Currency",
+                                "label": "Total expenditure on acquisition of Research and Development",
+                                "description": "Enter the full value (e.g. 56,234.33) or a value to the nearest £thousand (e.g. 56,000) Do not enter ‘56’ for £56,000",
+                                "min_value": {
+                                    "value": 0,
+                                    "exclusive": false
+                                },
+                                "decimal_places": 2,
+                                "currency": "GBP"
+                            }]
+                        }]
+                    },
+                    {
+                        "id": "block2437",
+                        "type": "Question",
+                        "questions": [{
+                            "id": "question2437",
+                            "title": "Did {{ metadata.ru_name }} invest in <em>acquisition of advanced machinery, equipment or software</em> for the purposes of current or future innovation?",
+                            "description": "",
+                            "type": "General",
+                            "answers": [{
+                                "id": "answer3179",
+                                "mandatory": false,
+                                "type": "Radio",
+                                "label": "",
+                                "description": "",
+                                "options": [{
+                                        "label": "Yes",
+                                        "value": "Yes"
+                                    },
+                                    {
+                                        "label": "No",
+                                        "value": "No"
+                                    }
+                                ]
+                            }]
+                        }],
+                        "routing_rules": [{
```

Comment:
this could be simpler..
```
                           {
                                "goto": {
                                    "block": "block2440",
                                    "when": [{
                                            "id": "answer3179",
                                            "condition": "not equals",
                                            "value": "No"
                                        }
                                    ]
                                }
                            },
                            {
                                "goto": {
                                    "block": "block2440",
                                    "when": [{
                                            "id": "answer3179",
                                            "condition": "not set",
                                        }
                                    ]
                                }
                            },
                            {
                                "goto": {
                                    "block": "block2438"
                                }
                            }
```

Reply:
This was done in Author and I have tried to not change what was done in Author. As mentioned in the PR desc, a lot of the rules could be simplified but I was not sure changing it all here was the right thing to do as it works, and how Author does these should be a separate discussion?

Reply:
Yes it should be raised as a separare discussion, but i can't see any reason not to change it here? as we learnt from LMS the more `when` blocks the slower it is

Reply:
Well, then author shouldn't add the rules if we are just gonna change it anyway?. But literally, every routing rule implemented by author can be simplified. Multiple surveys have gone in like this, we should have correct those as well if we want to be consistent.

I simply authors rules then.